.github/workflows/smoke-test.yml

```
@@ -200,6 +200,13 @@ jobs:
           sudo docker run --rm debian-hello
           sudo docker image rm debian-hello
 
+      # NOTE: regression test for check patch that forces
```

Indentation looks weird
```suggestion
       # NOTE: regression test for check patch that forces
```